src/hooks/users/useRecentLocationUsers.ts

```diff
@@ -1,10 +1,17 @@
-import { useMemo } from "react";
+import { isLoaded } from "react-redux-firebase";
+import { isEqual } from "lodash";
 
+import { User, userWithLocationToUser } from "types/User";
+
+import { WithId } from "utils/id";
+import { worldUsersSelector } from "utils/selectors";
 import { normalizeTimestampToMilliseconds } from "utils/time";
 
+import { useSelector } from "hooks/useSelector";
+import { useConnectWorldUsers } from "hooks/users";
 import { useUserLastSeenThreshold } from "hooks/useUserLastSeenThreshold";
 
-import { useWorldUsers } from "./useWorldUsers";
+const noUsers: WithId<User>[] = [];
```

Comment:
```suggestion
const noUsers: never[] = [];
```
?

Reply:
Nope, we want it to have the correct type.